src/cmd/sanity_test/export/onedrive.go

```diff
@@ -0,0 +1,86 @@
+package export
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	msgraphsdk "github.com/microsoftgraph/msgraph-sdk-go"
+
+	"github.com/alcionai/corso/src/cmd/sanity_test/restore"
+	"github.com/alcionai/corso/src/cmd/sanity_test/utils"
+	"github.com/alcionai/corso/src/internal/common/ptr"
+)
+
+func CheckOneDriveExport(
+	ctx context.Context,
+	client *msgraphsdk.GraphServiceClient,
```

Comment:
how about an `api.Client` here?

Reply:
Hmm, that might be cleaner. I was just [mimicing the existing code](https://github.com/alcionai/corso/blob/a973f69d182c161d6c30ad3376f43335f20b7c41/src/cmd/sanity_test/sanity_tests.go#L309). Let me do this change in a follow up PR though.